ajax/libs/html5-device-mockups/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "html5-device-mockups",
+  "filename": "device-mockups.min.css",
+  "description": "HTML5 mockups of popular devices, to showcase your portfolio and spice up your website.",
+  "keywords": [
+    "html5",
+    "device",
+    "mockups",
+    "demo",
+    "portfolio"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pixelsign/html5-device-mockups.git"
+  },
+  "homepage": "https://pixelsign.github.io/html5-device-mockups/",
+  "license": "MIT",
```

Comment:
On git the license is MIT, but on npm it is ICS... Please ask author about what license we should use here?Maybe both

Reply:
I have created a PR on the upstream repo trying to figure out which one is being used.
I am still waiting for their reply.

Reply:
@ralphchung It would be better to give the link of the PR you opened there. It helps reviewer trace the state of this pull request. So please give the link, thank you!

Reply:
@extend1994 It has been edited. Thanks!